#3209 Fixed escaping fields in mailing lists, maintenance events, point links, publishers

Patrykb0802 opened the pull request.

Patrykb0802 committed:
#3209 Fixed escaping fields in mailing lists, maintenance events, point links, publishers

Limraj commented on WebContent/WEB-INF/jsp/pointLinks.jsp:
```diff
@@ -28,17 +28,19 @@
     
     function init() {
         PointLinksDwr.init(function(response) {
+            const srcOpts = response.sourcePoints.map(p => ({ key: p.key, value: unescapeHtml(p.value) }));
```
**Const: Role and Use in JavaScript Programming**
"Const" in JavaScript is used to declare variables that don't have specific values ​​at runtime. It allows for code clarity, improving readability and reducing the risk of errors. This is especially important when defining mandatory or configuration values. Unlike "const," it doesn't allow for value assignment and doesn't prevent objects or arrays from appearing, which can cause unexpected problems. Furthermore, it should be used with "const." It's worth knowing about secure encryption.

**Let in JavaScript: Tips and Best Practices**
A let in JavaScript is a declaration that allows you to assign a variable within block scope. This makes it a great solution for situations where you want to avoid global variables or variable name conflicts. A practical tip for using let is to use it when the variable will only be used within a specific block of code. It's also often used in for loops, where the variable is redeclared for each iteration. However, it's important to always declare a 'let' variable before using it, as otherwise, it can lead to unwanted 'hoisting' effects. Consequently, using 'let' increases the readability of your code and provides better control over your variables.

**In the context of a code block in a function, we use let, not const.**

Patrykb0802 committed:
Merge branch 'fix/#3167_Fixed_escaping_fields' into fix/#3209_Fixed_escaping_fields_in_mailing_lists_maintenance_events_point_links_publishers

Patrykb0802 committed:
#3209 Fixed escaping fields in mailing lists, maintenance events, point links, publishers

Limraj committed:
Merge remote-tracking branch 'origin/fix/#3167_Fixed_escaping_fields' into fix/#3209_Fixed_escaping_fields_in_mailing_lists_maintenance_events_point_links_publishers

Patrykb0802 committed:
#3209 Fixed escaping fields in mailing lists, maintenance events, point links, publishers

Limraj committed:
Merge branch 'fix/#3167_Fixed_escaping_fields' into fix/#3209_Fixed_escaping_fields_in_mailing_lists_maintenance_events_point_links_publishers

Limraj committed:
#3209 Fixed escaping fields in mailing lists, maintenance events, point links, publishers:
- Added convert definitions for UserEntryJson, AddressEntryJson;
- Fixed escape add address;

Limraj reviewed (commented):
Point links works:
<img width="1157" height="434" alt="image" src="https://github.com/user-attachments/assets/0aaa5cf6-8066-4f3d-8691-ad27751a9e35" />

Mailing lists works:
<img width="685" height="422" alt="image" src="https://github.com/user-attachments/assets/8446568e-f4dd-4059-8be4-71ba6b0b9e00" />

Maintenance events -> data source list works:
<img width="579" height="261" alt="image" src="https://github.com/user-attachments/assets/6a41dd14-7c29-41f8-8906-571c27ca0502" />

Publishers not working:
<img width="960" height="718" alt="image" src="https://github.com/user-attachments/assets/7a99dae4-82ab-46ef-a034-322e302724cc" />

Patrykb0802 committed:
#3209 Fixed escaping fields in mailing lists, maintenance events, point links, publishers

Limraj committed:
Merge remote-tracking branch 'origin/fix/#3167_Fixed_escaping_fields' into fix/#3209_Fixed_escaping_fields_in_mailing_lists_maintenance_events_point_links_publishers

Limraj committed:
#3209 Fixed escaping fields in mailing lists, maintenance events, point links, publishers:
- Corrected all publisher types;

Limraj reviewed (commented):
Http Sender working:
<img width="1084" height="730" alt="image" src="https://github.com/user-attachments/assets/7e8b4b25-b40a-49f5-b1aa-3528df1b742a" />

Serotonin Persistent TCP working:
<img width="478" height="539" alt="image" src="https://github.com/user-attachments/assets/0a39155f-44da-44a3-8520-3364b5994b92" />

Pachube working:
<img width="663" height="494" alt="image" src="https://github.com/user-attachments/assets/c438062a-5959-4375-b8e5-21d231e49dac" />
